src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        return carDao.get(id);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        List<Driver> drivers = car.getDrivers();
+        if (!drivers.contains(driver)) {
+            drivers.add(driver);
+            car.setDrivers(drivers);
+        }
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        List<Driver> drivers = car.getDrivers();
+        drivers.remove(driver);
+        car.setDrivers(drivers);
```

Comment:
```suggestion
        car.getDrivers().remove(driver);
        carDao.update(car);
```
you should update car in the db after removing driver